lepotato/scripts/setup-user.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+USER=user
```

Comment:
Not specific to this particular recipe, but a general comment/idea: Would be great if the username was a template/variable that could easily be overridden on debos command line. ie. something like using `USER=${1:-user}` in script and then `scripts/setup-user.sh {{ $adminusername }}` in recipe action (plus possibly `{{ $adminusername := or .adminusername "user" }}` at the top). (Also general shell scripting hygene: use `set -e ; set -u` ?!)